pxtlib/packetio.ts

```diff
@@ -16,7 +16,7 @@ namespace pxt.packetio {
         isConnected(): boolean
         isConnecting(): boolean
         // flash the device, does **not** reconnect
-        reflashAsync(resp: pxtc.CompileResult): Promise<void>;
+        reflashAsync(resp: pxtc.CompileResult, progressCallback?: (percentageComplete: number) => void): Promise<void>;
```

Comment:
Where does this actually get implemented? I'm not seeing it in the PR but could just be missing something...

Reply:
Oh nvm looks like https://github.com/microsoft/pxt-microbit/pull/5200/files

Reply:
Ah yeah good point, would have been nice if I had added an annotation here; there is a packetio implementation here https://github.com/microsoft/pxt/blob/coreStartLoadingSpinner/pxtlib/hf2.ts#L185 which is the one used for arcade, I can look at making use of it there when we get back to arcade but arcade is still using the old pairing flow / didn't feel worth doubling (or quadrupling if I wanted to test the different arcade variants) for this pr / focused on micro:bit for now.